style.css

```diff
@@ -0,0 +1,111 @@
+Body {
```

Comment:
Codacy found an issue: [Expected "Body" to be "body" (selector-type-case)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)